bokehjs/src/coffee/models/glyphs/glyph.coffee

```diff
@@ -171,6 +172,14 @@ export class GlyphView extends View
 
     return result
 
+  _hit_rect_against_index: (geometry) ->
+    [x0, x1] = @renderer.xscale.v_invert([geometry.vx0, geometry.vx1])
+    [y0, y1] = @renderer.yscale.v_invert([geometry.vy0, geometry.vy1])
+    bb = hittest.validate_bbox_coords([x0, x1], [y0, y1])
```

Comment:
Without renaming this to `bb` (instead of `bbox`) another test fails with `bbox.empty` is not a function in `log_bounds` I don't understand why that should be the case, though.

Reply:
This is an effect of coffeescript's scoping rules. Don't reuse global names inside functions. Here https://github.com/bokeh/bokeh/pull/7008/commits/e865559a31ee7c3b132e7a235e5ecf3e8349b579 you have another example.

Reply:
Well, that's garbage